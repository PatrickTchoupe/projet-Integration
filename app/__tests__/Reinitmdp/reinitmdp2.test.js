/* eslint-disable import/imports-first */
/* eslint-disable import/no-unresolved */
/* eslint-disable no-undef */
/* eslint-disable import/no-extraneous-dependencies */
import 'react-native';
import React from 'react';
import ReinitMdp2 from './../../ReinitMdp/reinitMdpP2'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(<ReinitMdp2 />).toJSON();
    expect(tree).toMatchSnapshot();

});