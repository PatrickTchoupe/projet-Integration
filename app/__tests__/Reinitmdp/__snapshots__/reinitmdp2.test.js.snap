// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 13,
        "marginBottom": "3%",
        "marginLeft": "7%",
        "marginTop": "5%",
        "width": "90%",
      }
    }
  >
    Completez les champs suivants afin de réinitialiser votre mot de passe
  </Text>
  <Text
    style={
      Object {
        "fontSize": 12,
        "marginLeft": "7%",
        "marginTop": "2%",
        "textAlign": "center",
        "width": "90%",
      }
    }
  >
    Votre question secrète : 
  </Text>
  <Text
    style={
      Object {
        "fontSize": 12,
        "marginBottom": "5%",
        "marginLeft": "7%",
        "marginTop": "7%",
        "textAlign": "center",
        "width": "90%",
      }
    }
  >
     
     
  </Text>
  <Text
    style={
      Object {
        "fontSize": 12,
        "marginLeft": "7%",
        "marginTop": "2%",
        "textAlign": "center",
        "width": "90%",
      }
    }
  >
    Réponse secrète : 
  </Text>
  <TextInput
    allowFontScaling={true}
    maxLength={50}
    onChangeText={[Function]}
    placeholder="Entrez votre reponse secrete"
    rejectResponderTermination={true}
    style={
      Object {
        "borderColor": "#20232a",
        "borderRadius": 10,
        "borderWidth": 1,
        "fontSize": 12,
        "height": "10%",
        "marginBottom": "5%",
        "marginLeft": "5%",
        "marginTop": "5%",
        "width": "90%",
      }
    }
    underlineColorAndroid="transparent"
  />
  <Text
    style={
      Object {
        "fontSize": 12,
        "marginLeft": "7%",
        "marginTop": "2%",
        "textAlign": "center",
        "width": "90%",
      }
    }
  >
    Nouveau mot de passe : 
  </Text>
  <TextInput
    allowFontScaling={true}
    maxLength={50}
    onChangeText={[Function]}
    placeholder="Entrez votre nouveau mot de passe"
    rejectResponderTermination={true}
    secureTextEntry={true}
    style={
      Object {
        "borderColor": "#20232a",
        "borderRadius": 10,
        "borderWidth": 1,
        "fontSize": 12,
        "height": "10%",
        "marginBottom": "5%",
        "marginLeft": "5%",
        "marginTop": "5%",
        "width": "90%",
      }
    }
    underlineColorAndroid="transparent"
  />
  <Text
    style={
      Object {
        "fontSize": 12,
        "marginLeft": "7%",
        "marginTop": "2%",
        "textAlign": "center",
        "width": "90%",
      }
    }
  >
    Confirmez votre mot de passe : 
  </Text>
  <TextInput
    allowFontScaling={true}
    maxLength={50}
    onChangeText={[Function]}
    placeholder="Vérifiez votre nouveau mot de passe"
    rejectResponderTermination={true}
    secureTextEntry={true}
    style={
      Object {
        "borderColor": "#20232a",
        "borderRadius": 10,
        "borderWidth": 1,
        "fontSize": 12,
        "height": "10%",
        "marginBottom": "5%",
        "marginLeft": "5%",
        "marginTop": "5%",
        "width": "90%",
      }
    }
    underlineColorAndroid="transparent"
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "rgb(0, 150, 136)",
        "borderRadius": 20,
        "height": "16%",
        "marginLeft": "10%",
        "marginTop": "3%",
        "opacity": 1,
        "width": "80%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontSize": 30,
          "marginTop": "8%",
          "textAlign": "center",
        }
      }
    >
       Reinitialiser mot de passe 
    </Text>
  </View>
</View>
`;
